packages/wonder-blocks-card/src/components/card.tsx

```diff
@@ -184,9 +205,17 @@ const Card = React.forwardRef(function Card(
         elevation,
     });
 
+    const ariaLabelValue = labels?.cardAriaLabel
+        ? labels.cardAriaLabel
+        : ariaLabelledBy
+          ? undefined
+          : undefined;
```

Comment:
Could the `ariaLabelledBy ? undefined : undefined` be simplified to `undefined`? 😄